dynawo/sources/Models/CPP/ModelNetwork/DYNModelBus.cpp

```diff
@@ -23,6 +23,7 @@
 
 #include <boost/algorithm/string/predicate.hpp>
 
+#include "DYNNumericalUtils.h"
```

Comment:
I believe the modifications in the core part are still missing, no?

Reply:
what do you mean by core part?